src/page-modules/assistant/details/details-header/index.tsx

```diff
@@ -36,27 +36,57 @@ export default function DetailsHeader({ tripPattern }: DetailsHeaderProps) {
 
   const timeRange = `${departure} - ${arrival}`;
 
+  const isCancelled = tripPattern.legs.some(
+    (leg) => leg.fromEstimatedCall?.cancellation,
+  );
+
   return (
     <div className={style.container}>
-      <Typo.h2 textType="heading--big">
-        {fromName && toName
-          ? t(
-              PageText.Assistant.details.header.titleFromTo({
-                fromName,
-                toName,
-              }),
-            )
-          : t(PageText.Assistant.details.header.title)}
-      </Typo.h2>
+      <div className={style.headerContainer}>
+        {isCancelled && (
+          <ColorIcon
+            icon="status/Error"
+            className={style.situationIcon}
+            alt={t(PageText.Assistant.trip.tripPattern.isCancelled.label)}
+          />
+        )}
+        <Typo.h2 textType="heading--big">
+          {fromName && toName
+            ? t(
+                PageText.Assistant.details.header.titleFromTo({
+                  fromName,
+                  toName,
+                }),
+              )
+            : t(PageText.Assistant.details.header.title)}
+
+          {isCancelled &&
+            ` (${t(
+              PageText.Assistant.trip.tripPattern.isCancelled.title,
+            ).toUpperCase()})`}
+        </Typo.h2>
+      </div>
       <div className={style.tripDetails}>
         <div className={style.date}>
           <MonoIcon icon="time/Date" />
-          <Typo.p textType="body__primary">{weekdayAndDate}</Typo.p>
+          <Typo.p
+            textType={isCancelled ? 'body__primary--strike' : 'body__primary'}
```

Comment:
IIRC, in other places where we use strike-through to indicate that it is cancelled, the colour of the text is also changed. Should it be here as well? (I don't know how the contrast ratio is with a colour change.)

Reply:
You are correct. The secondary text color is used on the departure page to indicate a canceled departure. I talked to the product owner and he agreed that we could iterate on this in the next release. We should also talk to the designers at AtB do look into this when implementing it in the app